### PR TITLE
[TECHDEBT] Persistence Module, remove `DELETE` *actor* 'No-op' code

### LINKED ISSUE #226: [TECHDEBT] Utility Module, remove `DELETE` *actor* 'No-op' code
## Objective

Remove all `Delete Actor` related 'no-op' code.

## Origin Document

In the world of blockchain, nothing is ever truly deleted from the `world state`. In this case, the `Delete` operation is essentially a `no-op` because the structure always exists in history. 

Since it's clear that we don't have a use for `Delete` currently, the code should be removed until we have sufficient reason to have it.

## Goals

- Reduce code footprint 
- Reduce cognitive load
- 
## Deliverable

- Code complete implementation
- Updated CHANGELOG documentation

## General issue deliverables

- [ ] Update any relevant READMEs (local and/or global)
- [ ] Update any relevant global documentation & references
- [ ] If applicable, update the source code tree explanation
- [ ] If applicable, add or update a state, sequence or flowchart diagram using [mermaid](https://mermaid-js.github.io/mermaid/)

---

**Creator**: @andrewnguyen22

### PR DESCRIPTION
**Pull request by Olshansk**
## Description

See the description in https://github.com/pokt-network/pocket/pull/236.

## Issue

Fixes #226

## Type of change

- [ ] New feature, functionality or library
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Code health or cleanup
- [ ] Major breaking change
- [x] Documentation
- [ ] Other

## List of changes

- Removed no-op `DeleteActor` code in the persistence module
- Consolidated `CHANGELOG`s into one under `persistence/docs`
- Consolidated `README`s into one under `persistence/docs`

## Testing

NOTE: `make develop_test` and `LocalNet` are broken on `main` so this was validated via `make test_persistence`.

- [x] Validate there is no more delete functionality with `grep --exclude-dir='./vendor' -r -i " Delete"`
- [ ] `make test_persistence`
- [ ] `make test_all`
- [ ] [LocalNet](https://github.com/pokt-network/pocket/blob/main/docs/development/README.md) w/ all of the steps outlined in the `README`

## Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have tested my changes using the available tooling
- [ ] If applicable, I have made corresponding changes to related local or global README
- [ ] If applicable, I have added new diagrams using [mermaid.js](https://mermaid-js.github.io)
- [ ] If applicable, I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: persistence/docs/CHANGELOG.md [hunk 1]
```diff
@@ -7,31 +7,100 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
```
Comment (andrewnguyen22) [inline persistence/docs/CHANGELOG.md:8]: Please add `- Consolidated CHANGELOGs into one` to the PR description as well

File: persistence/docs/CHANGELOG.md [hunk 2]
```diff
@@ -7,31 +7,100 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 
-## [0.0.0.0] - 2022-08-08
+## [0.0.0.0] - 2022-09-22
```
Comment (andrewnguyen22) [inline persistence/docs/CHANGELOG.md:10]: If you're going to move / consolidate `changelog.md` to `persistence/docs/` then `README.md` should be moved as well

**Review by andrewnguyen22 — CHANGES_REQUESTED** [review]

**Comment by andrewnguyen22** [comment]
Just update PR to include full `list of changes` from changelog. Then it's good to go

**Review by andrewnguyen22 — CHANGES_REQUESTED** [review]
Just update PR to include full list of changes from changelog. Then it's good to go

**Comment by andrewnguyen22** [comment]
+ resolve conflicts  ^

**Comment by andrewnguyen22** [comment]
> Just update PR to include full list of changes from changelog. Then it's good to go

Can't approve until this is done

**Review by andrewnguyen22 — CHANGES_REQUESTED** [review]
> Just update PR to include full list of changes from changelog. Then it's good to go

Can't approve until this is done

**Review by andrewnguyen22 — APPROVED** [review]

**Comment by Olshansk** [comment]
NOTE: `make develop_test` and `LocalNet` are broken on `main` so this was validated via `make test_persistence`.

**Pull request merged**